src/ontology/components/cob-examples.owl

```diff
@@ -11,7 +11,6 @@
      xmlns:skos="http://www.w3.org/2004/02/skos/core#"
      xmlns:terms="http://purl.org/dc/terms/">
     <owl:Ontology rdf:about="http://purl.obolibrary.org/obo/cob/cob-examples.owl">
-        <owl:imports rdf:resource="http://purl.obolibrary.org/obo/cob/cob-edit.owl"/>
```

Comment:
@cmungall can you make sure this change does not break something? It resulted in various "imports not found" errors throughout the pipeline, but cant tell how important this is.